src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('First promise was rejected');
+  }, 3000);
+});
+
+const secondPromise = new Promise(resolve => {
+  body.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  body.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve('Second promise was resolved');
+  });
+});
+
+const leftClickPromise = new Promise(resolve => {
+  body.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const rightClickPromise = new Promise(resolve => {
+  body.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve();
+  });
+});
```

Comment:
these promises are not required, as you can use your `firstPromise` and `secondPromise` variables when handling `thirdPromise`